app-admin/haskell-updater/haskell-updater-1.3.2-r1.ebuild

```diff
@@ -3,12 +3,13 @@
 
 EAPI=8
 
+# ebuild generated by hackport 0.8.0.0.9999
+
 CABAL_FEATURES="nocabaldep"
 inherit haskell-cabal
 
 DESCRIPTION="Rebuild Haskell dependencies in Gentoo"
-HOMEPAGE="https://wiki.haskell.org/Gentoo#haskell-updater"
-SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+HOMEPAGE="https://github.com/gentoo-haskell/haskell-updater#readme"
```

Comment:
Not sure if that is because it's generated with hackport but i guess we don't need the extra linking to `#readme` in the `HOMEPAGE`